src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -64,51 +65,57 @@ function ButtonWithDropdownMenu<IValueType>({
             });
         }
     }, [windowWidth, windowHeight, isMenuVisible, anchorAlignment.vertical]);
-
     return (
         <View style={wrapperStyle}>
             {shouldAlwaysShowDropdownMenu || options.length > 1 ? (
                 <View style={[styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, style]}>
                     <Button
                         success={success}
                         pressOnEnter={pressOnEnter}
-                        ref={buttonRef}
-                        onPress={(event) => onPress(event, selectedItem.value)}
+                        ref={(ref) => {
+                            caretButton.current = ref;
```

Comment:
This PR caused a bug - [Tapping Pay button in an invoice crashes the app](https://github.com/Expensify/App/issues/44919#top)